tests/test_auth.py

```diff
@@ -18,6 +18,8 @@
 from portal.models.intervention import INTERVENTION
 from portal.models.role import ROLE
 from portal.models.user import (
+    LOCKOUT_PERIOD,
```

Comment:
Can you move these default/constant values to `config.py` in a subsequent PR to keep them consistent?

Reply:
Yes

Reply:
Created #2578 to track this.